plugin/audit_log/audit_log.c

```diff
@@ -785,6 +796,39 @@ int is_event_class_allowed_by_policy(unsigned int class,
   return (class_mask[policy] & (1 << class)) != 0;
 }
 
+static const CHARSET_INFO *charset_info= &my_charset_latin1;
+
+static
+const char *next_word(const char *str, size_t *len)
+{
+  while (*str && my_isspace(charset_info, *str))
+  {
+    if (*str == '/' && str[1] == '*' && str[2] == '!')
+      str+= 3;
+    else if (*str == '/' && str[1] == '*')
+    {
+      while (*str && !(*str == '*' && str[1] == '/'))
+        str++;
+    }
+    else
+      str++;
+  }
+
+  *len= 0;
+  while (str[*len] && my_isvar(charset_info, str[*len]))
```

Comment:
this patch works OK, here is what I see in the log:
```
"Query","<ID>","<DATETIME>","change_db","<CONN_ID>",0,"use `ąžąžąžą`","user1[user1] @ localhost []","localhost","","","ąžąžąžą"
```
I can't `use ąžąžąžą`, I need to ``use `ąžąžąžą` ``

What doesn't seem to work is disconnect event... and maybe connect as well. Looks like MySQL passing database wrong.
```
"Quit","<ID>","<DATETIME>","<CONN_ID>",0,"user1","user1","","","localhost","","ä…å¾ä…å¾ä…å¾ä…"
```
In this line database printed from the `mysql_event_connection`. Why is this happening?